client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -141,8 +141,12 @@ const CheckoutThankYou = React.createClass( {
 			const site = this.props.selectedSite.slug;
 
 			if ( purchases.some( isPlan ) ) {
-				page( `/plans/${ site }` );
-			} else if ( purchases.some( isDomainProduct ) || purchases.some( isDomainRedemption || purchases.some( isSiteRedirect ) ) ) {
+				page( `/plans/my-plan/${ site }` );
+			} else if (
+				purchases.some( isDomainProduct ) ||
+				purchases.some( isDomainRedemption ||
+				purchases.some( isSiteRedirect ) )
```

Comment:
Oops, sorry. I just noticed that it's actually `purchases.some( isDomainRedemption || purchases.some( isSiteRedirect ) )`. Could you please leave this on one line? That's more readable.